src/plugins/intel_gpu/tests/test_cases/non_max_suppression_test.cpp

```diff
@@ -96,10 +111,27 @@ struct non_max_suppression_basic : public testing::Test {
         return mem;
     }
 
+    format::type get_layout() {
+        return TypeWithLayout::layout;
+    }
+
+    static const data_types data_type = type_to_data_type<T>::value;
+
     const int pad = -1;
 };
 
-using nms_types = testing::Types<float, half_t>;
+using nms_types = testing::Types<TypeWithLayout<float, cldnn::format::bfyx>,
```

Comment:
this is a matter of taste) but I suggest using the following order

TypeWithLayout<float, cldnn::format::bfyx>,
TypeWithLayout<float, cldnn::format::bs_fs_yx_bsv16_fsv16>,
TypeWithLayout<float, cldnn::format::bs_fs_yx_bsv32_fsv32>,
TypeWithLayout<float, cldnn::format::b_fs_yx_fsv32>,
TypeWithLayout<float, cldnn::format::b_fs_yx_fsv16>,
TypeWithLayout<float, cldnn::format::bs_fs_yx_bsv32_fsv16>,

TypeWithLayout<half_t, cldnn::format::bfyx>,
TypeWithLayout<half_t, cldnn::format::bs_fs_yx_bsv16_fsv16>,
TypeWithLayout<half_t, cldnn::format::bs_fs_yx_bsv32_fsv32>,
TypeWithLayout<half_t, cldnn::format::b_fs_yx_fsv32>,
TypeWithLayout<half_t, cldnn::format::b_fs_yx_fsv16>,
TypeWithLayout<half_t, cldnn::format::bs_fs_yx_bsv32_fsv16>

so that you can immediately see which formats are supported

Reply:
agree

Reply:
done